### ISSUE TITLE
Some keyboard keys not working on input fields 

### ISSUE THREAD

**Issue by festim-deari** [SteveSandersonMS/WebWindow#27]
Arrow keys for navigation through the text do not work on input and textarea fields. Also "control + a" does not work. So there is no possibility for selecting or navigating through the text with keyboard only.

Tested on Windows 10.

**Comment by SQL-MisterMagoo** [comment]
This is a killer for anything that requires keyboard entry.

**Comment by SteveSandersonMS** [comment]
That's surprising. It must be an issue in the WebView2 library. Would you please consider reporting it there instead? https://github.com/MicrosoftEdge/WebViewFeedback

**Comment by Place1** [comment]
I've got the same issue on MacOS. I'm unable to use `cmd + <key>` meaning i can't copy/paste/selectall etc for an input element.

here's my sample code to reproduce:

```
public static void Main(string[] args)
        {
            var window = new WebWindows.WebWindow("My first WebWindow app");
            window.NavigateToString(@"
                <html>
                    <body>
                        <input>
                        <div id='output'></div>
                        <script>
                            var out = document.getElementById('output');
                            document.addEventListener('keyup', function(event) {
                                out.innerHTML = event.code;
                            });
                        </script>
                    </body>
                </html>
            ");
            window.WaitForExit();
        }
```
